### PR TITLE
Added conditional to DiagnosticContainer to show Question 

### PR DESCRIPTION
**Pull request by AdhamAH**
Screenshot 2020-03-24 at 00 42 02
depending on the state
Screenshot 2020-03-24 at 00 42 26
Screenshot 2020-03-24 at 00 42 51

### CODE REVIEW

**Comment by liadove** [comment]
Good job!

**Comment by AdhamAH** [comment]
> Good job!

like the app lol

File: client/src/components/DiagnosticContainer.js [hunk 1]
```diff
@@ -39,6 +43,7 @@ const dateOptions = {
 };
 
 function DiagnosticContainer(props) {
+  const [healthLogged] = useState(false);
```
Comment (friedger) [inline client/src/components/DiagnosticContainer.js]: This should come from redux, shouldn't it?

Comment (liadove) [reply client/src/components/DiagnosticContainer.js]: Yes, it will once we know what redux would look like. For now, it's just for condition underneath

Comment (SomeMoosery) [reply client/src/components/DiagnosticContainer.js]: We should still get state from Redux whenever possible as it should be the single source of truth for application state, even if it's just placeholder state for now

**Comment by AdhamAH** [comment]
As seen from the comments, no need for it.

**Pull request closed without merging**